cli/src/pcluster/utils.py

```diff
@@ -223,9 +223,7 @@ def log_stack_failure_recursive(stack_name, failed_states=None, indent=2):
                 # parallelcluster-fsx-fail-FSXSubstack-65ITLJEZJ0DQ/
                 # 3a4ecf00-51e7-11ea-8e3e-022fd555c652 was not successfully created:
                 # The following resource(s) failed to create: [FileSystem]."
-                substack_error = re.search(
-                    ".+/({0}.+)/".format(PCLUSTER_STACK_PREFIX), event.get("ResourceStatusReason")
-                )
+                substack_error = re.search(".+ (arn:aws:cloudformation[^ ]+) ", event.get("ResourceStatusReason"))
```

Comment:
I think we should get rid of this `if event.get("ResourceType") == "AWS::CloudFormation::Stack":` code. It was for substacks and we don't have them anymore.